packages/oui-collapsible/src/collapsible.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-collapsible" data-ng-class="{'oui-collapsible_open':$ctrl.expanded}">
+    <div class="oui-collapsible__header" data-ng-click="$ctrl.toggle()">
+        <div class="oui-collapsible__header-text" data-ng-bind="::$ctrl.title"></div>
+        <button
+            type="button"
+            aria-label="{{::$ctrl.ariaLabel}}"
+            aria-pressed="{{$ctrl.expanded}}"
+            class="oui-collapsible__toggle-button">
+            <i class="oui-icon oui-icon-chevron-down oui-collapsible__toggle-icon" aria-hidden="true"></i>
+        </button>
+    </div>
+    <div class="oui-collapsible__body" data-ng-transclude></div>
```

Comment:
Accessibility issue: we have no `aria-expanded` and `aria-controls`.
Please see [here](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region) for more informations :)